ticketing/eventbrite.py

```diff
@@ -20,7 +20,7 @@
     import_transaction_message,
     sync_off_instructions,
 )
-from ticketing.brown_paper import (
+from ticketing.sync_user_utils import (
```

Comment:
Note to self - someday, make a ticket importer class for common functionality like this, then just inherit it.